### ISSUE TITLE
Feature Request : District-wise Recovered and death data

### ISSUE THREAD

**Issue by vivek50** [covid19india/api#114]
Add District-wise Number of Recovered and death data in [v2/state_district_wise.json](https://api.covid19india.org/v2/state_district_wise.json)
that will be more helpful.

**Comment by sudevschiz** [comment]
Yes, we can add this information by using the District field in the Death and Recovered sheet. However, the data that is coming out is not very clear about where the recoveries happen.

Will keep this open enhancement issue for somebody to work on.

**Comment by Arusara** [comment]
@sudevschiz i would like to work on this item. can you please assign to me?

**Comment by sudevschiz** [comment]
Assigned to you. The districts of deceased and recovered cases are sparse. So, please discuss here how you will represent this information here. Thank you!

**Comment by Arusara** [comment]
@sudevschiz i am planing to add death and recovered  field in json 

      {
        "district": "Alappuzha",
        "confirmed": 5,
        "lastupdatedtime": "",
        "delta": {
          "confirmed": 0
        }
        "recovered":3,
        "deceased":0
      },

**Comment by sudevschiz** [comment]
Thank you. Awaiting PR :)

**Comment by Arusara** [comment]
@sudevschiz PR is raised. please check once 
https://github.com/covid19india/api/pull/132

**Comment by vivek50** [comment]
Thank you to implement this feature.
